cesium/internal/domain/writer.go

```diff
@@ -80,12 +85,13 @@ func Write(ctx context.Context, db *DB, tr telem.TimeRange, data []byte) error {
 type Writer struct {
 	alamos.Instrumentation
 	WriterConfig
-	prevCommit telem.TimeStamp
-	idx        *index
-	fileKey    uint16
-	internal   xio.TrackedWriteCloser
-	presetEnd  bool
-	closed     bool
+	prevCommit  telem.TimeStamp
+	idx         *index
+	fileKey     uint16
+	internal    xio.TrackedWriteCloser
+	presetEnd   bool
+	lastPersist telem.TimeStamp
```

Comment:
I would rename this variable to `lastIndexPersist` or something. That makes it clearer that we're tracking index persistence instead of some other form of general persistence.